ote/src/cljs/ote/views/front_page.cljs

```diff
@@ -16,8 +16,30 @@
             [ote.db.modification :as modification]
             [ote.time :as time]
             [stylefy.core :as stylefy]
-            [ote.style.base :as style-base]))
+            [ote.style.base :as style-base]
+            [reagent.core :as r]))
 
+(defn- delete-service-action [e! {::t-service/keys [id name]
+                                  :keys [show-delete-modal?]
+                                  :as service}]
+  [:span
+   [ui/icon-button {:on-click #(e! (ts/->DeleteTransportService id))}
+    [ic/action-delete]]
+   (when show-delete-modal?
+     [ui/dialog
+      {:open true
+       :title (tr [:dialog :delete-transport-service :title])
+       :actions [(r/as-element
+                  [ui/flat-button
+                   {:label (tr [:buttons :cancel])
+                    :primary true
+                    :on-click #(e! (ts/->CancelDeleteTransportService id))}])
+                 (r/as-element
+                  [ui/flat-button {:label (tr [:buttons :delete])
```

Comment:
Regardless of the original material ui example I would suggest that Save/Delete/Ok buttons should be raised-buttons in modals.